.gitignore

```diff
@@ -28,3 +28,4 @@ spec/dummy/tmp/
 .rspec
 .rvmrc
 .rbenv-version
+Gemfile.lock
```

Comment:
Please read this article. http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/

TDLR; Gemfile.lock should only be checked in for projects, not gems.